style.css

```diff
@@ -24,55 +29,88 @@ body {
 }
 
 h1 {
-  color: rgb(28, 16, 108);
-  font-size: 30px;
+  color: #091e42;
+  padding-top: 40px;
+  padding-left: 60px;
+  font-size: 48px;
   font-family: Arial, Helvetica, sans-serif;
-  align-items: flex-end;
-  margin-left: 10px;
+  line-height: 60px;
```

Comment:
- [ ] Please use the correct font following the requirements of the project, as required by the previous reviewer. The Figma provides all the information of the correct font as shown below:
<img width="1277" alt="Screen Shot 2022-08-19 at 8 17 13 AM" src="https://user-images.githubusercontent.com/47673311/185608380-bf773101-ce89-4509-b611-769c72cb598f.png">

- [ ] You have to use Poppins font. Please check the correct size, color, etc. Follow the instructions to set up the font correctly on your app [here](https://fonts.google.com/specimen/Poppins).